photonai/optimization/config_grid.py

```diff
@@ -47,12 +47,12 @@ def create_global_config_grid(pipeline_elements, add_name=''):
     for i in global_hyperparameter_list:
         total_product_num = total_product_num * len(i)
     if total_product_num > threshold:
-        warn_text = 'The entire configuration grid entails more than ' + str(threshold) + ' possible configurations. ' \
-                                                                                          'This might take veeeeeeery ' \
-                                                                                          'looooong to both compute ' \
-                                                                                          'and process.'
-        logger.warning(warn_text)
-        raise Warning(warn_text)
+        msg = 'The entire configuration grid entails more than ' + str(threshold) + ' possible configurations. ' \
+                                                                                    'This might take very ' \
+                                                                                    'long to both compute ' \
+                                                                                    'and process.'
+        logger.error(msg)
+        raise ValueError(msg)
```

Comment:
why did this become a value error? should be warning?

Reply:
serious answer? tried with warnings: test for that case runs forever...a timebound did not directly work. 
Since we raise a warning beforehand, I thought nobody would miss these great grids.